src/cvl/Makefile

```diff
@@ -0,0 +1,86 @@
+################################################################################
+#                                                                              #
+#  Copyright 2019 Broadcom. The term Broadcom refers to Broadcom Inc. and/or   #
+#  its subsidiaries.                                                           #
+#                                                                              #
+#  Licensed under the Apache License, Version 2.0 (the "License");             #
+#  you may not use this file except in compliance with the License.            #
+#  You may obtain a copy of the License at                                     #
+#                                                                              #
+#     http://www.apache.org/licenses/LICENSE-2.0                               #
+#                                                                              #
+#  Unless required by applicable law or agreed to in writing, software         #
+#  distributed under the License is distributed on an "AS IS" BASIS,           #
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.    #
+#  See the License for the specific language governing permissions and         #
+#  limitations under the License.                                              #
+#                                                                              #
+################################################################################
+
+all: precheck deps schema tests
+GO?=/usr/local/go/bin/go
+SRC_FILES=$(shell find . -name '*.go' | grep -v '_test.go' | grep -v '/tests/')
+TEST_FILES=$(wildcard *_test.go)
+TOP_DIR := $(abspath ../..)
+GOFLAGS:=
+BUILD_DIR:=build
+GO_DOWNLOAD_PATH:=$(BUILD_GOPATH)
+CVL_PKG=$(TOP_DIR)/pkg/linux_amd64/cvl.a
+
+CVL_TEST_DIR = $(TOP_DIR)/build/tests/cvl
+CVL_TEST_BIN = $(CVL_TEST_DIR)/cvl.test
+
+ifdef DEBUG
+	GOFLAGS += -gcflags="all=-N -l"
+endif
+
+precheck:
+	$(shell mkdir -p $(BUILD_DIR))
+
+deps:  $(BUILD_DIR)/.deps $(CVL_PKG) $(CVL_TEST_BIN)
+
+	
+$(BUILD_DIR)/.deps:
+# Patch code
+	@grep ParseJsonMap  $(GO_DOWNLOAD_PATH)/src/github.com/antchfx/jsonquery/node.go || \
+	printf "\nfunc ParseJsonMap(jsonMap *map[string]interface{}) (*Node, error) {\n \
```

Comment:
Could this be done as a patch instead? Like the ygot.patch and goyang.patch patches in the main Makefile.

Reply:
This has been fixed by Partha.